blog/models.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    class Meta:
+        verbose_name = "user"
+        verbose_name_plural = "users"
+
+    def __str__(self):
+        return f"{self.username}"
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(User,
+                              on_delete=models.CASCADE,
+                              related_name="posts")
+    title = models.CharField(max_length=200)
+    content = models.TextField()
```

Comment:
Similar to the `title` field, you might want to add `blank=False, null=False` to the `content` field to ensure that the content of the Post is provided and it can't be empty.